suite-native/module-send/src/components/SendFeesForm.tsx

```diff
@@ -47,27 +51,60 @@ type SendFeesNavigationProps = StackToStackCompositeNavigationProps<
 
 const DEFAULT_FEE = 'normal';
 
-export const SendFeesForm = ({ accountKey, tokenContract, feeLevels }: SendFormProps) => {
+export const SendFeesForm = ({ accountKey, tokenContract }: SendFormProps) => {
     const navigation = useNavigation<SendFeesNavigationProps>();
 
     const account = useSelector((state: AccountsRootState) =>
         selectAccountByKey(state, accountKey),
     );
+    const feeLevels = useSelector(selectFeeLevels);
+
+    const networkFeeInfo = useSelector((state: FeesRootState) =>
+        selectNetworkFeeInfo(state, account?.symbol),
+    );
+
+    const normalFeeLevelInfo = networkFeeInfo?.levels.find(l => l.label === 'normal');
+    const networkType = account?.symbol ? getNetworkType(account.symbol) : undefined;
+
+    const minimalFeeLimit =
```

Comment:
can you do only  `feeLevels?.normal?.estimatedFeeLimit` ?

Reply:
The `feeLevels.normal` might be of a type that does not include `estimatedFeeLimit`. So the Typescript shouts that we are trying to access a prop that might not be present.